lib/kafka-consumer.js

```diff
@@ -21,6 +21,49 @@ var DEFAULT_CONSUME_LOOP_TIMEOUT_DELAY = 500;
 var DEFAULT_CONSUME_TIME_OUT = 1000;
 util.inherits(KafkaConsumer, Client);
 
+var eagerRebalanceCallback = function(err, assignment) {
+  // Create the librdkafka error
+  err = LibrdKafkaError.create(err);
+  // Emit the event
+  self.emit('rebalance', err, assignment);
```

Comment:
`self` is not defined here

Reply:
Oh, I use it with a custom rebalance callback in my code and did not hit there.